src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2722,7 +2765,11 @@ namespace UnitarySystems {
             state.dataSize->UnitaryHeatCap = EqSizing.DesHeatingLoad;
 
         } else {
-            state.dataSize->UnitaryHeatCap = this->m_DesignHeatingCapacity;
+            if (state.dataSize->CurZoneEqNum == 0 || !isVarSpeedCoolCoil) {
+                state.dataSize->UnitaryHeatCap = this->m_DesignHeatingCapacity;
+            } else {
+                EqSizing.HeatingCapacity = false;
+            }
```

Comment:
To be clear, I believe this needs to be like this but am weary of diffs. UnitarySystem always sets the supp heater size = heating coil size. For HPs this is the same size as the cooling coil (until now). For non-HPs this is the same size as the heating coil which should be sized to meet only the heating load. The above works for the defect file and causes no diffs, but ... Maybe I should try this while its fresh in our minds?

    if (this->m_DehumidControlType_Num == DehumCtrlType::CoolReheat) {
        set same size as cooling coil
    } else {
        let heating load dictate size regardless of HP configuration (I was offered a 5, 10 or 15 kW strip heat for my 3 ton HP (10 kW))
    }

Reply:
This looks more general than ` if (state.dataSize->CurZoneEqNum == 0 || !isVarSpeedCoolCoil) {`. But the CoolReheat block needs to set to the max of cooling coil or heating load.

Reply:
This still works for the UnitarySystem file. Am running unit tests now. Anything stand out?

            if (this->m_DehumidControlType_Num == DehumCtrlType::CoolReheat) {
                state.dataSize->UnitaryHeatCap = max(this->m_DesignCoolingCapacity, this->m_DesignHeatingCapacity);
            } else {
                EqSizing.HeatingCapacity = false;
            }

Reply:
No, other than I don't understand why the else isn't `state.dataSize->UnitaryHeatCap = this->m_DesignHeatingCapacity;`

Reply:
Well, I guess it could be. Resetting EqSizing.HeatingCapacity lets the heating coil size normally instead of using the parent data.

Reply:
No, that's the way it was before.

    state.dataSize->UnitaryHeatCap = this->m_DesignHeatingCapacity;